PowerEditor/installer/nativeLang/danish.xml

```diff
@@ -116,23 +116,23 @@ The comments are here for explanation, it's not necessary to translate them.
                     <Item id="42086" name="Dato tid (tilpasset)"/>
                     <Item id="42008" name="Forøg linieindrykning"/>
                     <Item id="42009" name="Formindsk linieindrykning"/>
-                    <Item id="42010" name="Dupliker aktuel linie"/>
+                    <Item id="42010" name="Duplikér aktuel linie"/>
                     <Item id="42079" name="Fjern ens linier"/>
                     <Item id="42077" name="Fjern fortløbende duplikerede linier"/>
                     <Item id="42012" name="Opdel linier"/>
                     <Item id="42013" name="Sammenføj linier"/>
                     <Item id="42014" name="Flyt aktuel linie op"/>
                     <Item id="42015" name="Flyt aktuel linie ned"/>
-                    <Item id="42059" name="Sortér linier leksikografisk (stigende)"/>
-                    <Item id="42060" name="Sortér linier leksikografisk (faldende)"/>
-                    <Item id="42080" name="Sortér linier leksikografisk (stigende/ignorer versaler)"/>
-                    <Item id="42081" name="Sortér linier leksikografisk (faldende/ignorer versaler)"/>
-                    <Item id="42061" name="Sortér linier som heltal (stigende)"/>
-                    <Item id="42062" name="Sortér linier som heltal (faldende)"/>
-                    <Item id="42063" name="Sortér linier som decimaltal med komma (stigende)"/>
-                    <Item id="42064" name="Sortér linier som decimaltal med komma (faldende)"/>
-                    <Item id="42065" name="Sortér linier som decimaltal med punktum (stigende)"/>
-                    <Item id="42066" name="Sortér linier som decimaltal med punktum (faldende)"/>
+                    <Item id="42059" name="Sortér linier alfabetisk (A > Å)"/>
+                    <Item id="42060" name="Sortér linier alfabetisk (Å > A)"/>
+                    <Item id="42080" name="Sortér linier alfabetisk (a > å, ignorér versaler)"/>
+                    <Item id="42081" name="Sortér linier alfabetisk (å > a, ignorér versaler)"/>
+                    <Item id="42061" name="Sortér linier som heltal (0 > ∞)"/>
+                    <Item id="42062" name="Sortér linier som heltal (∞ > 0)"/>
+                    <Item id="42063" name="Sortér linier som decimaltal med komma (0 > ∞)"/>
+                    <Item id="42064" name="Sortér linier som decimaltal med komma (∞ > 0)"/>
+                    <Item id="42065" name="Sortér linier som decimaltal med punktum (∞ > 0)"/>
```

Comment:
Is this line correkt?? I guess "0 > infinity" is the right here.

Reply:
The translation of "42062" - "42065" is "Sorting Lines" Ascending or Descending. there's no meaning of infinity (∞)

Reply:
Oke 👍